lib/containers/Hotels.js

```diff
@@ -117,8 +158,8 @@ class Hotels extends React.Component {
             }
             contentMore={
               <TextGroup>
-                <div className={'content-more text-extended'} dangerouslySetInnerHTML={{ __html: hotel.accommodationExtended }} />
-                <div className={'content-more'} dangerouslySetInnerHTML={{ __html: hotel.accommodationMore }} />
+                <div className="content-more text-extended" dangerouslySetInnerHTML={{ __html: hotel.accommodationExtended }} />
```

Comment:
Why do you change theses className attributes?

Reply:
i didn't changed the class names i did only what the IDE told me: ESLint complained about the curly brace - because it is not necessary.